.github/workflows/test.yaml

```diff
@@ -139,8 +191,15 @@ jobs:
       run: |
         ${{ matrix.tox-wrapper }} tox ${{ matrix.trial-target }}
 
+    # If one of the above steps fails, fire up tmate for remote debugging.
+    - name: Tmate debug on failure
+      if: ${{ !cancelled() && env.TMATE_DEBUG == 'yes' }}
```

Comment:
The workflow for using this action, then, will be to change `TMATE_DEBUG` at the top of the file to `'yes'` and re-run the build, then change it back to `'no'` again afterwards?

I guess there's a good reason not to have it turned on all the time?  https://github.com/mxschmitt/action-tmate#manually-triggered-debug looks like a slightly nicer workflow, maybe it's not too much more work to do it that way?

Reply:
For me, editing the text was easier than using the Github action manual workflow trigger.

It is true that you can forget to set it back to `no` afterward.

-------

You don't want it turned on all the time, as then on failure it will "hang" and wait for someone to connect via SSH.

----------

I will push a commit that will also add manual triggers.
Not sure what are the permissions for manual triggers... that is, if you can trigger from a fork.